website/src/auth.tsx

```diff
@@ -82,6 +134,7 @@ export const UserProvider = (props: { children: any }) => {
 
         console.log("Login success. Result: ", data)
         alert("Login successful")
+        navigate("/")// TODO navigate to dashboard
```

Comment:
Perhaps we need to fork here based on if the user is verified and possibly send them to the `auth/confirm-signup` page if they are not.

Reply:
AWS will give us a `UserNotConfirmedException` if a user tries to login but their account is not confirmed. Ive also added some nicer error handling for when this error does appear.

Relevant docs: https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_InitiateAuth.html#API_InitiateAuth_Errors